### ISSUE TITLE
cscope: cannot read list size from file cscope.out

### ISSUE THREAD

**Issue by Remington012** [portante/pycscope#46]
I realize this is the same error message as issue #4 , but it's still keeping me from using `pycscope`.  

After putting the (restricted) list of python files I want to index into `./cscope.files`, and then successfully using `/usr/local/bin/pycscope -i ./cscope.files`  to generate `./cscope.out`, I'm still unable to use the generated index:
```
$ cscope -d -i ./cscope.files -f ./cscope.out
cscope: cannot read list size from file ./cscope.out
$ du -sh cscope.out
25M     cscope.out
$ cscope -V
cscope: version 15.8a
$ 
```
This is with an older Ubuntu system.  I do find multiple references of other people [hitting this issue](https://stackanswers.net/questions/cscope-cscope-error-cscope-cannot-read-list-size-from-file-cscope-out) recently-ish, so it can't just by my system.  The complaints I've found are about paths containing spaces and about the index being "too large", both of which seem like solvable issues...?

I tried using `pycscope` on a much smaller subset of python files (making sure none had spaces), yielding a `cscope.out` index of less than 1 MB, and sure enough now it works.  However, I know I've successfully used `cscope` in C and C++ projects where the `cscope.out` index was much larger, so this still seems like an issue where `pycscope` isn't correctly generating the index once it gets "large enough".

**Comment by Remington012** [comment]
I did some more testing, where I progressively trimmed-down the list of python files being indexed.  I was surprised to learn that the size of the generated `cscope.out` index really does need to be under 1 MB (for less than 30k lines of python code) before it would stop hitting this issue.

**Comment by Remington012** [comment]
Interestingly, this effect seems to be sensitive to the _CONTENTS_ of those python files.  With some more exploratory testing, I was able to generate a 10 MB `cscope.out` index over python files in other subtrees (~480k lines of python code), where the issue did **NOT** occur.

**Comment by kuator** [comment]
Same error here

**Comment by wanghaoxu001** [comment]
I tried to use cscope command line tool like
 `$ cscope -d -f cscope.out -R -L2 _backup_file`                                                                                                                    
and got this:
`cscope: cannot read list size from file cscope.out`
The cscope.out is about 1.9MB.

**Comment by ClayShoaf** [comment]
I'm also getting this error. I installed pycscope using pyenv to install it using `python 3.8.10`. I thought maybe there was some problem with my system python being 3.10. I have no idea what a normal cscope.out file is supposed to look like, so I don't know if there is a problem with that file or if the problem lies with cscope.

**Comment by ClayShoaf** [comment]
I tried again in a different spot and it worked.

From this repo: https://github.com/lightningd/plugins

The `./summary` folder produces the error when I open `summary.py`, and the `./noise` directory works fine when I open `noise.py`

**Comment by penvirus** [comment]
Also got the same error in some of repositories.  Just realized it is an offset issue.

See https://github.com/portante/pycscope/pull/50/commits/84730d476eb8e80494daa30d4a6ff7c481632054 for the proposed fix.

**Comment by portante** [comment]
Fixed by #50.

**Comment by ClayShoaf** [comment]
> Fixed by #50.

Thank you all so much! I can finally navigate python code within vim and figure out what the hell is going on for the first time ever. Oh man, this is such a great feeling!
